components/Editor.js

```diff
@@ -81,6 +83,25 @@ class Editor extends React.Component {
       loading: false
     }
 
+    const storedThemes = getThemes(localStorage)
+
+    newState.themes = storedThemes
+      ? [...storedThemes, ...this.state.themes]
```

Comment:
I believe you can just do `[...storedThemes, ...this.state.themes]` without the ternary 👍